python/pysrc/kaskada/_timestream.py

```diff
@@ -31,14 +31,18 @@
 if TYPE_CHECKING:
     import graphviz
 
-#: A literal value that can be used as an argument to a Timestream operation.
+# docstring must be under the definition :(
 LiteralValue: TypeAlias = Optional[Union[int, str, float, bool, timedelta, datetime]]
 
```

Comment:
If so, then let's not have a space between definition and docstring (here and below)